cmd/oceantv/broadcast_machine.go

```diff
@@ -164,6 +166,24 @@ func (sm *broadcastStateMachine) handleGoodHealthEvent(event goodHealthEvent) er
 	return nil
 }
 
+func (sm *broadcastStateMachine) handleControllerFailureEvent(event controllerFailureEvent) error {
+	sm.log("handling controller failure event")
+	switch sm.currentState.(type) {
+	case *directStarting:
+		onFailureClosure(sm.ctx, sm.ctx.cfg, true)(fmt.Errorf("Controller not reporting. Likely caused by low voltage."))
+		sm.transition(newDirectIdle(sm.ctx))
+	case *vidforwardPermanentStarting:
+		onFailureClosure(sm.ctx, sm.ctx.cfg, true)(fmt.Errorf("Controller not reporting. Likely caused by low voltage."))
+		sm.transition(newVidforwardPermanentSlate())
```

Comment:
Maybe we should just go to vidforwardPermanentIdle considering it hasn't even been able to start yet.

Reply:
Done.